extensions/binding/org.eclipse.smarthome.binding.bluetooth/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="bluetooth"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Bluetooth Binding</name>
+    <description>This is the binding for Bluetooth Smart.</description>
```

Comment:
Well, both of these are of course bluetooth - bluetooth v 4.0 :) However maybe `ble` does fit the binding better...